src/Files.App/ViewModels/UserControls/ToolbarViewModel.cs

```diff
@@ -787,6 +807,7 @@ public async Task SetAddressBarSuggestions(AutoSuggestBox sender, IShellPage she
 
 					if (sender.Text.StartsWith(">"))
 					{
+						IsCommandPaletteOpen = true;
```

Comment:
Shouldn't `IsCommandPaletteOpen` be set to `false` if `sender.Text.StartsWith(">")` is false?

Reply:
Great. I added this logic to the else branch.